docs/app/components/navigation-bar-search/navigation-bar-search.component.ts

```diff
@@ -130,28 +146,34 @@ export class NavigationBarSearchComponent {
         // if search query changes reset the active index
         this.activeIdx = 0;
 
-        // store the current search query
-        this.query = value;
+        if (value === null || value === '') {
+
+            // If the query is empty, show last 5 selected results.
+            this.results = this.history;
 
-        // if the search query is empty or less than 3 characters show no results
-        if (this.query === null || this.query === '' || this.query.length < 3) {
+        } else if (value.length < QUERY_MIN_CHARS) {
+
+            // Show nothing for wide searches
             this.results = [];
-            return;
-        }
 
-        // get the latest results
-        this.results = this.data.filter((item: ISearchResult) => {
-            return item.link.title.toLowerCase().indexOf(this.query.toLowerCase()) !== -1;
-        });
+        } else {
+
+            // get the latest results
+            this.results = this.data.filter((item: ISearchResult) => {
```

Comment:
I notice if I type in the name of a section with subsections of the same name eg. dashboard, im getting duplicate results, we should probably filter them out?

![image](https://user-images.githubusercontent.com/20795331/27277764-3f3b8e50-54d7-11e7-914a-ae29d8fe3928.png)